src/main/plugin/dcat2/update-fixed-info.xsl

```diff
@@ -236,6 +236,32 @@
     <xsl:namespace name="gml" select="'http://www.opengis.net/gml/3.2'"/>
   </xsl:template>
 
+  <xsl:template name="add-resource-type">
+    <xsl:variable name="inScheme" select="'https://registry.geonetwork-opensource.org/dcat/type'"/>
+    <xsl:variable name="typeURI">
+      <xsl:choose>
+        <xsl:when test="name() = 'dcat:Dataset'">
+          <xsl:value-of select="concat($inScheme, '/', 'dataset')"/>
+        </xsl:when>
+        <xsl:when test="name() = 'dcat:DataService'">
+          <xsl:value-of select="concat($inScheme, '/', 'service')"/>
+        </xsl:when>
+      </xsl:choose>
+    </xsl:variable>
+    <xsl:variable name="thesaurusKey" select="'external.theme.dcat-type'"/>
+
+    <dct:type>
+      <skos:Concept rdf:about="{$typeURI}">
```

Comment:
Maybe here at some point we should also be able to set using rdf:resource or skos:Concept. Not sure how we can handle that @josegar74 Maybe a global flag in update-fixed-info ? and then depending on the record we apply the same strategy all over the places ? 
But we can start with that.